stylus/components/wizard.styl

```diff
@@ -241,8 +241,8 @@ $wizard-desc--footer
 
 $wizard-header-fixed
     position fixed
-    top rem(14)
-    left 0
+    top rem(8)
+    left rem(16)
```

Comment:
I do not understand why there are "magic" values here. It seems we are moving the header where in fact we want to increase the padding of the arrow.